roles/scylla-collectd-client/tasks/main.yaml

```diff
@@ -1,5 +1,17 @@
+- name: Gather facts
+  action: ec2_facts
+
+- set_fact: node="scylla-{{ansible_hostname }}"
```

Comment:
That is alright, that was only a stylish nitpick.

I do have another comment on this now that I am using it: this makes collectd see use the private IP of the instance as an identifier. That is quite cryptic, since this is not readily available at the EC2 panel by default.

Couldn't we use the instance name (which has the added benefit of being usually even shorter) ?